<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Prot/48.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditLogin">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>31</y>
      <width>235</width>
      <height>96</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: 1px solid rgb( 192, 192, 192);
 border-radius: 3px; 
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>372</y>
      <width>122</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 245, 245);</string>
    </property>
    <property name="text">
     <string>Войти через</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPass">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>180</y>
      <width>312</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-image: url(:/Prot/TextEdit.png);
border: 1px solid rgb( 192, 192, 192);
 border-radius: 3px; 
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>321</y>
      <width>312</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Prot/TextEdit.png);
border: 1px solid rgb( 192, 192, 192);
 border-radius: 3px; 
</string>
    </property>
    <property name="text">
     <string>Авторизация</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>280</y>
      <width>13</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 245, 245);
</string>
    </property>
    <property name="text">
     <string>Запомнить</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_exit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_hide">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>31</y>
      <width>235</width>
      <height>96</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPass_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>237</y>
      <width>312</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-image: url(:/Prot/TextEdit.png);
border: 1px solid rgb( 192, 192, 192);
 border-radius: 3px; 
}
 border-radius: 3px; </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>418</y>
      <width>152</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Prot/TextEdit.png);
 border-radius: 3px; </string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>418</y>
      <width>156</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Prot/TextEdit.png);
 border-radius: 3px; </string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
